CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 20220411
+
+- ecs version: 1.61.0-1
```

Comment:
Should we add these two?
```
source al2022 ami: al2022-ami-minimal-2022.0.20220315.0-kernel-5.15-x86_64
distribution al2022 release: 2022.0.20220315
```

Reply:
yes good point, since it's the first al2022 I will add those